### DISCUSSION TITLE
jobs.lookup_jid is too verbose if cmd.script was used - filtering options please?

### DISCUSSION THREAD

**Discussion by ScriptJunky** [saltstack/salt#67266] [Feature Requests]
```
salt -G roles:webservers cmd.script salt://scripts/open.java.versions --async
```

Calling the output when using CLI will always produce stuff that we don't really care to see at the point in time (pid, retcode, stderr);

```
-0-root@saltmaster:/srv/salt/scripts # salt-run jobs.lookup_jid 20200122041547835372
webserver001:
    ----------
    pid:
        30126
    retcode:
        0
    stderr:
    stdout:
        Running Java: oracle_jdk-1.8.0_11
webserver002:
    ----------
    pid:
        32282
    retcode:
        0
    stderr:
    stdout:
        Running Java: oracle_jdk-1.8.0_11
```

Could a way to filter this output be added to jobs.lookup_jid?

**Comment by xeacott** [comment]
So instead something like this... where only stdout is shown?
```
-0-root@saltmaster:/srv/salt/scripts # salt-run jobs.lookup_jid 20200122041547835372

webserver001:
    ----------
    Running Java: oracle_jdk-1.8.0_11

webserver002:
    ----------
    Running Java: oracle_jdk-1.8.0_11

```
I actually don't know maybe using a different out-putter can achieve this but otherwise it would be a feature request. But, I'll ping @saltstack/team-core in case there already is something built in to achieve this. :)

**Comment by ScriptJunky** [comment]
Yes that exactly, or some means to filter like maybe;

```--filter stdout```
or 
```--filter stderr,pid```

That would allow a bit more granular control as some folks might want something other than stdout, or more than just stdout but not everything....

**Comment by sagetherage** [comment]
Discussed with team-core 1/24/2020 at this time the core team will keep this in the backlog as lower priority and the community is welcome to submit a PR or use as suggested ^^ to use a filter.
